### DISCUSSION TITLE
Help...noob cant find new integration

### DISCUSSION THREAD

**Discussion by zdavesf** [sebr/bhyve-home-assistant#114] [Q&A] [answered]
Running Home Assistant Core v2022.3.3 (Via Unraid Docker)
- I successfully downloaded V2.2.1 integration via HACS and restarted HA
![bhyve hacs](https://user-images.githubusercontent.com/99004176/157807424-022199f8-d01e-4bbf-8313-5aabcc1e1edc.PNG)

- I then added the following to my configuration.yaml    (changed the username and password to my actual orbit account)
![bhyve configuration yaml](https://user-images.githubusercontent.com/99004176/157807639-ee4e0a29-4158-44f2-8f87-64fbc1513535.png)

-restarted HA from browser
-in HA goto configuration --> Devices and Services --> +Add integration, type in bhyve, or orbit and nothing is found
![bhyve integration](https://user-images.githubusercontent.com/99004176/157807847-bb3275d1-5706-4198-a16c-c21c988b2fbb.PNG)

What am I doing wrong?

**Comment by zdavesf** [comment]
updated my config.yaml to match exactly as in the read me with the !secret and then update the secret.yaml file.... still i do not seem any entities under developer tools
![secret password](https://user-images.githubusercontent.com/99004176/158006053-35ad2c40-5f3a-4ec3-877e-036ab50bf802.PNG)
![config with secret](https://user-images.githubusercontent.com/99004176/158006055-4d865582-4e48-4c90-b265-7c68dabb22ce.PNG)


log is attached
[home-assistant.log](https://github.com/sebr/bhyve-home-assistant/files/8237137/home-assistant.log)
.

**Reply by sebr** [reply]
The logs don't contain any meaningful lines relating to the BHyve integration. Is it possible that the `logger` yaml config was added after this log file was created? Perhaps you can restart and try again?

**Comment by zdavesf** [comment, marked as answer]
sebr thank you for the reply.  I noticed that the log was useless and I slowed down and did some thinking. Turns out that I actually was not editing my config/secrets file through windows explorer navigating to my unraid server file path.  Apparently the permissions were not allowing me to make the change.  after a bunch of searching the quick and easy way was to make a copy of my config.yaml and secrets.yaml through windows explorer, edit them, delete the originals and rename the new ones to match the original.

while i should do more searching and figure out the "chown" command, the copy/paste method got me to finally see the entities. I am very happy (and very embarrased, obvious newb here)  thank you for your paitence.

![orbit integration](https://user-images.githubusercontent.com/99004176/159105124-57b8214a-6c89-43ca-b620-38b942c48d45.PNG)
